server/controllers/questions.js

```diff
@@ -0,0 +1,55 @@
+const {
+  get, post, report, markHelpful,
+} = require('../models/questions');
+
+module.exports = {
+  getQuestions(req, res) {
+    const productId = req.query.product_id;
+    return get(productId)
+      .then((data) => {
+        res.status(200).send(data.rows); // likely need to restructure data for client
```

Comment:
by default express sends a status code of 200, so probably not necessary here.

Reply:
Oo good to know, thank you!